dpgen2/op/collect_data.py

```diff
@@ -99,6 +111,34 @@ def execute(
         Path(name).mkdir()
         if mixed_type:
             ms.to_deepmd_npy_mixed(name)  # type: ignore
+            if use_ele_temp:
```

Comment:
it is better to write a function for adding ele_temp. 
perhaps we should firstly fix dpdata...